service/src/controllers/quiz.ts

```diff
@@ -20,16 +20,16 @@ const addQuiz = async (
     name: string,
     description: string,
     duration: number,
-    startTime: Date,
```

Comment:
can we make all of these non- compulsory (add a `?` after all of them for `addQuiz`) to avoid merge conflicts